WordPress/Classes/ViewRelated/NUX/Login.storyboard

```diff
@@ -134,78 +134,78 @@
                         <rect key="frame" x="0.0" y="0.0" width="383" height="676"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <stackView opaque="NO" contentMode="scaleToFill" verticalHuggingPriority="245" axis="vertical" spacing="10" translatesAutoresizingMaskIntoConstraints="NO" id="ZBL-pW-jTv" userLabel="Wrapper Stack View">
-                                <rect key="frame" x="0.0" y="64" width="383" height="612"/>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="ozJ-hT-OEw">
```

Comment:
> we've ended up with several levels of nested views,

That's a good point.  I went back and removed some of the nesting in the controllers that had text entry.